lib/dialects/abstract/index.js

```diff
@@ -42,7 +42,14 @@ AbstractDialect.prototype.supports = {
   migrations: true,
   upserts: true,
   constraints: {
-    restrict: true
+    restrict: true,
+    addConstraint: true,
```

Comment:
Yes, Only MSSQL & postgres support complete `ADD CONSTRAINT` , `DROP CONSTRAINT` SQL keywords. MySQL supports `ADD CONSTRAINT` but there is no `DROP CONSTRAINT`. Sqlite doesn't support both of them. As @mickhansen suggested above, i used dialect.feature check instead of dialect.name checks to skip/allow tests